packages/tooltip/src/Tooltip.tsx

```diff
@@ -115,6 +114,28 @@ interface PopoverFunctionParameters {
 
 type ModifiedPopoverProps = Omit<PopoverProps, 'active' | 'refEl'>;
 
+type PortalProps = OneOf<
+  {
+    /**
+     * Specifies that the popover content will appear portaled to the end of the DOM,
+     * rather than in the DOM tree.
+     *
+     * default: `true`
```

Comment:
nit: Can we use @default just for consistency

Reply:
From what I see, we don't use this consistently right now, but while "@" holds semantic meaning in JSDoc, it doesn't in TSDoc. In fact, from what I can tell, it may be `@defaultValue`: https://github.com/microsoft/tsdoc/issues/27

I'd recommend that we stick with `default:`, or move entirely to `@defaultValue` rather than trying to use semantically-meaningless `@default`.

Reply:
It'd be nice to at least follow one convention, whether it be JSDoc or TSDoc, rather than us invent our own.

Reply:
I agree. The thing is that we've already "invented" the convention, and use it in the majority of cases (according to my search of the repo). If we're gonna change conventions, we should move to one supported by TSDoc, and do it everywhere IMO.

Reply:
FWIW, there wasn't a tag in TSDoc for defaults when we started using the convention, so I don't think we were wrong in using this in the past.

Reply:
Yeah for sure. Sounds like a good tech debt cleanup ticket.